string name
float64 commanded
float64 position
float64 velocity
float64 current
float64 temperature
int32 active
int32 zeroed
